package.json

```diff
@@ -72,7 +73,7 @@
     "jspdf": "https://github.com/parallax/jsPDF.git#b7a1d8239c596292ce86dafa77f05987bcfa2e6e",
     "jszip": "^3.10.1",
     "kolibri-constants": "^0.2.0",
-    "kolibri-design-system": "~3.0.0",
+    "kolibri-design-system": "https://github.com/AlexVelezLl/kolibri-design-system#2ecd75b2a96a71f4fce5a6f1c7a6092f06f6547f",
```

Comment:
@AlexVelezLl could you please point the KDS version to 3.1.0?

Reply:
Updated!